woof-code/rootfs-skeleton/usr/bin/pupx

```diff
@@ -146,6 +146,12 @@ if [ -f ~/.Xwaylandrc ]; then
         sed -i "s%^XWAYLAND_SCREENSAVER_DELAY=.*%XWAYLAND_SCREENSAVER_DELAY=%" ~/.Xwaylandrc
 	fi
     exit
+elif [ -f "$XDG_CONFIG_HOME/swayidle/config" ]; then # created by set_wl_monsleep at startup absent
```

Comment:
WAYLAND_DISPLAY is set when running labwc, so you'll never reach this `elif` - is that right?

Reply:
See line 7 of the diff

Reply:
BTW< I tested with replacing every instance of `,Xwaylandrc` with a dummy and it works as expected.. you get an unchecked checkbox which you need to check to sensitize and change the spinbutton value.